cfgtool

```diff
@@ -285,14 +285,34 @@ Stay safe. Use crypto. Don´t drink and factorize."
 
 }
 
+#http://blog.pragbits.com/it/2015/02/09/slack-notifications-via-curl/
+__notify_slack() {    # format message as a code block ```${msg}```
+    SLACK_TITLE=$1
+    SLACK_BODY=$2
+    SLACK_URL="https://hooks.slack.com/services/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"
+    curl -X POST --data "payload={
+     \"attachments\":[
+        {
+           \"color\":\"#00ff00\",
+           \"fields\":[
+              {
+                 \"title\":\"${SLACK_TITLE}\",
```

Comment:
The json will be malformed if SLACK_TITLE or SLACK_BODY contain ". Please use some quoting/escaping tool.